### ISSUE TITLE
Doesn't work on M1 / ARM

### ISSUE THREAD

**Issue by fr3fou** [meilisearch/meilisearch-kubernetes#90]
I tried deploying the Helm chart on a minikube cluster on my M1 MacBook and got the following error:
```sh
simo at makro in ~/C/p/m/charts
↪ kubectl logs meilisearch-0
Error: An internal error has occurred. `Function not implemented (os error 38)`.
```

**Environment (please complete the following information):**
- OS: macOS 11.5.2 on MacBook Pro 2021 M1
- MeiliSearch version: v0.24.0
- meilisearch-kubernetes version: v0.1.21

**Comment by alallema** [comment]
Hi @fr3fou,
Indeed we don't have yet an official docker image for M1. But it should be available on the next release v0.25.0!

**Comment by alallema** [comment]
The new docker image for M1 is finally out and available! You can check how it's work [here](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)
